scripts/install-maria.sh

```diff
@@ -1,14 +1,15 @@
 
 # Check If Maria Has Been Installed
 
+set -e
+set -x
```

Comment:
Possibly, I find it helpful when I can see which command exactly failed, if something fails. But happy to remove it if others think it's too verbose.